.github/workflows/nightly.yml

```diff
@@ -105,28 +72,15 @@ jobs:
             i=$((i+1))
           done < <(printf '%s\n' "$TEST_WALLET_SIGNING_KEYS")
 
-      - name: Run Marlowe Scaling on Preview
+      - name: Run Marlowe Benchmark on Preview
         run: |
-          nix run .#marlowe-scaling -- \
-            --marlowe-runtime-host marlowe-runtime-preview.scdev.aws.iohkdev.io \
-            2 \
-            "$(cat test-wallets/test.0.addr)=test-wallets/test.0.skey" \
-            "$(cat test-wallets/test.1.addr)=test-wallets/test.1.skey" \
-            "$(cat test-wallets/test.2.addr)=test-wallets/test.2.skey" \
-            "$(cat test-wallets/test.3.addr)=test-wallets/test.3.skey" \
-            "$(cat test-wallets/test.4.addr)=test-wallets/test.4.skey" \
-            "$(cat test-wallets/test.5.addr)=test-wallets/test.5.skey" \
-            "$(cat test-wallets/test.6.addr)=test-wallets/test.6.skey" \
-            "$(cat test-wallets/test.7.addr)=test-wallets/test.7.skey" \
-            "$(cat test-wallets/test.8.addr)=test-wallets/test.8.skey" \
-            "$(cat test-wallets/test.9.addr)=test-wallets/test.9.skey" \
-            "$(cat test-wallets/test.10.addr)=test-wallets/test.10.skey" \
-            "$(cat test-wallets/test.11.addr)=test-wallets/test.11.skey" \
-            "$(cat test-wallets/test.12.addr)=test-wallets/test.12.skey" \
-            "$(cat test-wallets/test.13.addr)=test-wallets/test.13.skey" \
-            "$(cat test-wallets/test.14.addr)=test-wallets/test.14.skey" \
-            "$(cat test-wallets/test.15.addr)=test-wallets/test.15.skey" \
-            "$(cat test-wallets/test.16.addr)=test-wallets/test.16.skey" \
-            "$(cat test-wallets/test.17.addr)=test-wallets/test.17.skey" \
-            "$(cat test-wallets/test.18.addr)=test-wallets/test.18.skey" \
-            "$(cat test-wallets/test.19.addr)=test-wallets/test.19.skey"
+          mkfifo node.socket
```

Comment:
This is superfluous, `socat` will create its own unix socket (which is not the same as a fifo)

Reply:
Makes sense. Thanks!